devtools-ci-index.yaml

```diff
@@ -3012,15 +3012,15 @@
             git_organization: redhat-developer
             git_repo: rh-che
             ci_project: 'devtools'
-            ci_cmd: 'HOST_URL="che.prod-preview.openshift.io" /bin/bash ./functional-tests/devscripts/prepare_env.sh && /bin/bash ./functional-tests/devscripts/run_tests.sh'
+            ci_cmd: 'HOST_URL="che.prod-preview.openshift.io" /bin/bash source ./functional-tests/devscripts/prepare_env.sh && /bin/bash ./functional-tests/devscripts/run_tests.sh'
```

Comment:
I think  `/bin/bash source` won't as it'll try to find the `source` executable and it's not an executable.

I would replace it with:

```
ci_cmd: 'export HOST_URL="che.prod-preview.openshift.io" && source ./functional-tests/devscripts/prepare_env.sh && ./functional-tests/devscripts/run_tests.sh'
```

Reply:
Ok thank you for your advice - I'll make new commit.